.travis.yml

```diff
@@ -6,10 +6,9 @@ env:
   - FORMULA=omero BREW_OPTS=--only-dependencies
 before_script:
     - brew update
-    - brew tap ome/alt
     - brew tap homebrew/science
+    - brew tap ome/alt file:///$(pwd)
```

Comment:
Should `file:///` be `file://` given that you'll get an extra `/` from the output of `$(pwd)`?